.sops.yaml

```diff
@@ -6,8 +6,6 @@
 keys:
   - &steveej 6F7069FE6B96E894E60EC45C6EEFA706CB17E89B
   - &age_steveej age1tkvtkw62xy90xc5xdcq836wgyrwlwmdslh76cete5g98vvvhj34qvwdw0g
-  - &jost-s D299483493EAE6B2B3D892B6D33548FA55FF167F
```

Comment:
Never used to encrypt anything and we should prefer age over GPG, setting up GPG is much harder and not documented in the project.